Configure

```diff
@@ -501,6 +502,9 @@ my @disable_cascades = (
 
     "crypto-mdebug"     => [ "crypto-mdebug-backtrace" ],
 
+    # scrypt requires PBKDF2
+    "pbkdf2"            => [ "scrypt" ],
```

Comment:
Hmmm, I do think it's a good idea to be able to strip down the code basis to the bare minimum (ultimately maybe one day have the ability to have a OpenSSL version that's stripped down so much it can conveniently run on very small embedded systems, maybe even in PSK-only DTLS or something along the lines). That said, it does increase complexity. Since it's a completely separate commit, it's your judgement if you want to take it mainline or not. I think there's good arguments to be made for either choice, so no hurt feelings :-)